packages/components/src/components/popover/popover.tsx

```diff
@@ -67,7 +67,7 @@ export class Popover {
   public connectedCallback(): void {
     applyConstructableStylesheetStyles(this.host, getSlottedAnchorStyles);
     addDocumentEventListener(this);
-    this.isNativePopoverCase = detectNativePopoverCase(this.host, false);
+    this.isNativePopoverCase = !!detectNativePopoverCase(this.host, false);
```

Comment:
boolean casting shouln't be necessary as `detectNativePopoverCase` returns a boolean